modules/persistent-storage-csi-google-cloud-file-create-sc.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc
+
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-google-cloud-file-create-sc_{context}"]
+= Creating a storage class for Google Filestore Storage
+
+After installing the operator, you should create a storage class for dynamic provisioning of Google Filestore volumes. 
+
+.Prerequisites
+* You are logged in to the running {product-title} cluster.
+
+.Procedure
+To create a storage class:
+
+. Create a storage class using the following example YAML file:
++
+[source,yaml]
+.Example YAML file
+--
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: my-sc
+  uid: edabf3f4-7bdc-4d95-b682-4307d83327b2
+  resourceVersion: '55980'
+  creationTimestamp: '2022-08-10T17:26:28Z'
+  managedFields:
+    - manager: Mozilla
+      operation: Update
+      apiVersion: storage.k8s.io/v1
+      time: '2022-08-10T17:26:28Z'
+      fieldsType: FieldsV1
+      fieldsV1:
+        'f:allowVolumeExpansion': {}
+        'f:parameters':
+          .: {}
+          'f:network': {}
+        'f:provisioner': {}
+        'f:reclaimPolicy': {}
+        'f:volumeBindingMode': {}
+provisioner: filestore.csi.storage.gke.io
+parameters:
+  network: network-name <1>
+reclaimPolicy: Retain
+allowVolumeExpansion: true
+volumeBindingMode: WaitForFirstConsumer
+--
+<1> Specify the name of the network that you want to use. 
+
+. Specify the name of the network you want to use.
++
+It is recommended to specify the network that the Filestore instances should be created in. If no network is specified, the Container Storage Interface (CSI) driver tries to create the instances in the default network of the project.
+On IPI installations, the network name is typically the cluster name with the suffix "-network". However, on UPI installations, the network name can be any value chosen by the user.
++
+You can find out the network name by inspecting the `MachineSets` objects with the following command: 
++
+[source, command]
+----
+$ oc -n openshift-machine-api get machinesets -o yaml | grep "network:"
+            - network: google-filestore-6fb4h-network
```

Comment:
I recommend replacing `google-filestore-6fb4h-network` with something less random, like `google-filestore-network`.

Also, I believe it's better to add a sentence explaining that the network name in this cluster is `google-filestore-network`.